post_installation_configuration/cluster-tasks.adoc

```diff
@@ -508,6 +508,20 @@ You can create a `MachineSet` object to host only infrastructure components. You
 include::modules/infrastructure-components.adoc[leveloffset=+2]
 include::modules/nodes-scheduler-node-selectors-cluster.adoc[leveloffset=+2]
 
+[id="assigning-machine-set-resources-to-infra-nodes"]
+== Assigning machine set resources to infrastructure nodes
+
+After creating an infrastructure machine set, the `worker` and `infra` roles are applied to new infra nodes. Nodes with the `infra` role are not counted toward the total number of subscriptions that are required to run the environment, even when the `worker` role is also applied.
+
+However, with an infra node being assigned as a worker, there is a chance user workloads could get inadvertently assigned to an infra node. To avoid this, you can apply a taint to the infra node and tolerations for the pods you want to control.
```

Comment:
Just a thought on the first sentence, up to you.

"However, when an infra node is assigned the `worker` role, there is a chance that user workloads can get assigned inadvertently to the infra node."